go.mod

```diff
@@ -447,30 +447,28 @@ require (
 // for Go build tools to accept the `go.mod`.
 replace (
 	github.com/facebookincubator/nvdtools => github.com/stackrox/nvdtools v0.0.0-20231111002313-57e262e4797e
-	// we need https://github.com/fullsailor/pkcs7/pull/42 to be merged
-	github.com/fullsailor/pkcs7 => github.com/stackrox/pkcs7 v0.0.0-20220914154527-cfdb0aa47179
-	github.com/gogo/protobuf => github.com/connorgorman/protobuf v1.2.2-0.20240207122816-e936d453291c
 
 	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.0.0-20230714151239-78b1f5f70b8a
 
-	// github.com/mikefarah/yaml/v2 is a clone of github.com/go-yaml/yaml/v2.
-	// Both github.com/go-yaml/yaml/v2 and github.com/go-yaml/yaml/v3 do not provide go.sum
-	// so dependabot is not able to check dependecies.
-	// See https://github.com/go-yaml/yaml/issues/772
-	// Therefore we point all to our fork of `go-yaml` - github.com/stackrox/yaml/v2|v3
-	// where we provide the actual `go.sum`.
-	github.com/mikefarah/yamlg/v2 => gopkg.in/yaml.v2 v2.4.0
-
 	github.com/nxadm/tail => github.com/stackrox/tail v1.4.9-0.20210831224919-407035634f5d
 
 	// github.com/stackrox/helm-operator is a modified fork of github.com/operator-framework/helm-operator-plugins that
 	// we currently depend on.
 	github.com/operator-framework/helm-operator-plugins => github.com/stackrox/helm-operator v0.0.12-0.20230825152000-1361e2f7db46
 
-	github.com/sigstore/cosign/v2 => github.com/stackrox/cosign/v2 v2.0.0-20231206143943-626efb842da7
-
 	github.com/tecbot/gorocksdb => github.com/DataDog/gorocksdb v0.0.0-20200107201226-9722c3a2e063
```

Comment:
Probably can be removed together with rocksdb. If not, we probably can _just_ use DataDog's one, since it is still maintained.

Reply:
- https://github.com/stackrox/stackrox/pull/10390

Reply:
Let's leave it untouched as it should be removed soon